crates/languages/src/python/highlights.scm

```diff
@@ -221,3 +243,33 @@
   "match"
   "case"
 ] @keyword
+
+; Definition keywords def, class, async def, lambda
+[
+  "async"
+  "def"
+  "class"
+  "lambda"
+] @keyword.definition
+
+((identifier) @attribute.builtin
+  (#match? @attribute.builtin "^(classmethod|staticmethod|property)$"))
+
+; Builtin types as identifiers
+[
+  (call
+    function: (identifier) @type.builtin)
+  (call
+    arguments: (argument_list
+    (identifier) @type.builtin))
+  (call
+    arguments: (argument_list
+      (keyword_argument
+        value: (identifier) @type.builtin)))
+  (type (identifier) @type.builtin)
+  ; also check if type binary operator left identifier for union types
+  (type
+    (binary_operator
+      left: (identifier) @type.builtin))
+  (#match? @type.builtin "^(bool|bytearray|bytes|complex|dict|float|frozenset|int|list|memoryview|object|range|set|slice|str|tuple)$")
```

Comment:
```suggestion
(#any-of? @type.builtin "bool" "bytearray" "bytes" "complex" "dict" "float" "frozenset" "int" "list" "memoryview" "object" "range" "set" "slice" "str" "tuple")
```
This isn't going to use a regex and is probably a bit cleaner.

Reply:
Applied your suggestion